paths/graphql/06-mutation.md

```diff
@@ -10,6 +10,8 @@ sidebar:
   nav: "github-graphql"
 main-content: |
 
+  ![img](../images/gifs/graphql/mutation.gif)
```

Comment:
If I could get some eyes on this change from @brianamarie (since we were talking about it before the weekend started) that would be amazing.